cmd/plan_test.go

```diff
@@ -88,3 +89,45 @@ func (suite *PlanTestSuite) TestGetPriorInstalledPackages_NoPreinstalledPackages
 func installedPackagesAreEqual(expected, actual desc.Desc) bool {
 	return expected.Package == actual.Package && expected.Version == actual.Version && expected.Repository == actual.Repository
 }
+
+func CopyDir(fs afero.Fs, src string, dst string) error {
```

Comment:
should compare this against implementations such as https://github.com/otiai10/copy/blob/master/copy.go

Reply:
I took a look. Our implementation doesn't handle symlinks, but overall they seem to be doing the same basic thing.

Since this function was recently demoted to a test-helper, and it has proven sufficient for the test cases that use it (most of which would always fail if this function didn't work), I think it's safe to leave it.